config/manifests/bases/falcon-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: falcon-operator.v0.0.0
+  namespace: placeholder
```

Comment:
Another example of `placeholder` namespace.

Reply:
Per the doc, using `placeholder` here is super common. See https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md

Reply:
OK. Resolving the placeholder comments.